app/forms/participant_mentor_form.rb

```diff
@@ -13,9 +13,7 @@ def mentor
   end
 
   def available_mentors
-    User.where(
-      id: school.mentor_profiles_for(Cohort.find(cohort_id)).select(:user_id),
-    ).order(:full_name)
+    SchoolCohort.find_by(school_id: school_id, cohort_id: cohort_id).mentors.order(:full_name)
```

Comment:
Will this include withdrawn mentors?

Reply:
I don't know. Maybe. It feels like there are a bunch of tests missing.

Reply:
Turns out yes, it was. Good spot.